### ISSUE TITLE
[Tracking] Threads Bugs

### ISSUE THREAD

**Issue by grgia** [Expensify/App#19877]
## Tracking Issue for Threads-Related Bugs

Here are the current open threads-related bugs/improvements/fixes from v1 implementation:

**Bugs:**
[Hovering over a user's icon in thread should show First/Last name over email if it exists](https://github.com/Expensify/App/issues/19709)
[Hide threads from LHN if they don't have a comment yet](https://github.com/Expensify/App/issues/19205)
[3Eyed todd appears when opening a thread for the first time](https://github.com/Expensify/App/issues/19045)
[Edits/Reactions on parent action in child report does not send the proper onyx notification](https://github.com/Expensify/Expensify/issues/282991)
[Non-Room Workspace chats are failing when creating threads](https://github.com/Expensify/App/issues/19554)
[LHN- Thread messages chat for own messages do not appear for other user in LHN](https://github.com/Expensify/App/issues/19858)
[Removed user still able to reply to threads](https://github.com/Expensify/App/issues/19686)
[LHN logic for threads is wrong (inconsistent with other chats)](https://github.com/Expensify/App/issues/19447)
[Reply in thread count is not update while send message faster](https://github.com/Expensify/App/issues/19445)
[There is a noticeable lag before chat turns from Unread to Read in the LHN for IOU reports/threads](https://github.com/Expensify/App/issues/19444)
[When clicking on Reply in thread "No activity yet" displayed briefly in LHN](https://github.com/Expensify/App/issues/19349)
[Editing parent comment in the thread are not reflected until the thread is revisited](https://github.com/Expensify/App/issues/19348)
[On the first message of a thread the added reactions appear only after page refresh](https://github.com/Expensify/App/issues/19238)
[Removed user still able to reply to threads](https://github.com/Expensify/App/issues/19686)

**Improvements:**
[Use HOC in HeaderView and ReportActionItem](https://github.com/Expensify/App/issues/18769) 
[Remove type-specific avatars from rooms and threads](https://github.com/Expensify/App/issues/19264) 
[DRY parent & child params in CreateReportAction and UpdateReportComment](https://github.com/Expensify/Expensify/issues/287384)

**Comment by grgia** [comment]
closing in favor of tags in Threads Project View

**Comment by bondydaa** [comment]
oh I had just added https://github.com/Expensify/App/issues/19686 to this list, is there a tag I should put on it?
